src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/transformers/MethodCallTransformer.kt

```diff
@@ -95,16 +94,16 @@ internal class MethodCallTransformer(
         storeLocal(argumentsArrayLocal)
         loadLocal(argumentsArrayLocal)
         invokeStatic(Injections::onMethodCall)
-        
+
         val deterministicMethodDescriptor = newLocal(OBJECT_TYPE)
         storeLocal(deterministicMethodDescriptor)
-        
+
         val deterministicCallIdLocal = newLocal(LONG_TYPE)
         pushDeterministicCallId(deterministicMethodDescriptor)
         storeLocal(deterministicCallIdLocal)
-        
+
         invokeBeforeEventIfPluginEnabled("method call $methodName", setMethodEventId = true)
-        
```

Comment:
We should add a linter to our development cycle leter. So that it at least deals with the left tabulations automatically